crm_compassion/models/res_partner.py

```diff
@@ -31,7 +31,7 @@ def open_events(self):
         }
 
     @api.multi
-    def make_ambassador(self):
+    def create_odoo_user(self):
```

Comment:
It's not the method in python that must be renamed but the action string which is somwhere in a xml file. But you can as well rename the code method, but then please update as well in the xml of the action, where this method is called, otherwise it will fail!